tensorboard/plugins/mesh/README.md

```diff
@@ -0,0 +1,111 @@
+# Mesh Plugin
+
+## Overview
+Mesh and point cloud are important and powerful types of data to represent 3D
+shapes and widely studied in the field of computer vision and computer 
+graphics.
+3D data is becoming more ubiquitous and researchers challenge new problems like
+3D geometry reconstruction from 2D data, 3D point cloud semantic segmentation,
+aligning or morphing 3D objects and so on. Therefore, visualizing results,
+especially during the training stage, is critical to better understand how
+the model performs.
+
+![Mesh Plugin in TensorBoard](
+https://storage.googleapis.com/tensorflow-graphics/git/readme/tensorboard_plugin.jpg)
+
+This plugin intends to display 3D point cloud or mesh (triangulated point 
+cloud)
+in TensorBoard. In addition, it allows the user to
+interact with the rendered object.
+
+## Summary API
+Either a mesh or a point cloud can be represented by a set of tensors. For
+example, one can viewpoint cloud as a set of 3D coordinates of the points and
+some colors associated with each point.
+
+```python
+from tensorboard.plugins import mesh
+...
+point_cloud = tf.constant([[[0.19, 0.78, 0.02], ...]], shape=[1, 1064, 3])
+point_colors = tf.constant([[[128, 104, 227], ...]], shape=[1, 1064, 3])
+
+summary = mesh.summary('point_cloud', vertices=point_cloud, colors=point_colors)
+```
+
+NOTE `colors` tensor is optional in this case but can be useful to show
+different semantics of the points.
+
+The plugin currently supports only triangular meshes which are different from
```

Comment:
As I understood it, the plugin supports both bare point clouds and
triangulated meshes (just not n-gon meshes). But this paragraph suggests
that bare point clouds aren’t supported. Consider rephrasing to
something like

```markdown
A mesh can be represented by a point cloud together with a set of faces,
where each face is represented by a list of indices of vertices in
clockwise order around the face. This plugin currently only supports
triangular faces. Construct a mesh by including the `faces` parameter to
`mesh.summary`:
```

if this is correct (I’m just guessing about your orientation choice in
particular).